proxmox/virtual_environment_vm.go

```diff
@@ -130,29 +157,188 @@ func (c *VirtualEnvironmentClient) GetVMStatus(nodeName string, vmID int) (*Virt
 	return resBody.Data, nil
 }
 
+// MoveVMDisk moves a virtual machine disk.
+func (c *VirtualEnvironmentClient) MoveVMDisk(nodeName string, vmID int, d *VirtualEnvironmentVMMoveDiskRequestBody) error {
+	taskID, err := c.MoveVMDiskAsync(nodeName, vmID, d)
```

Comment:
HTTP 500 error "you can't move to the same storage with same format" is returned here